website/src/content/help-needed.mdx

```diff
@@ -147,15 +147,14 @@ When a Custom Application state is changed from **Ready** to **Draft**, it is un
 
 </Info>
 
-## Project access is not automatically updated for newly created Projects
+## Project access is not automatically updated for newly created Projects on AWS
 
 When a new Project is created in the Organization, the Project cannot be accessed despite the **Project access** setting being configured to **Install for all projects of this organization** for an installed Custom Application.
 
 To grant access, [the Project access must be updated manually](https://docs.commercetools.com/merchant-center/managing-custom-applications#updating-projects-access) for the installed Custom Application.
 
-## Custom Applications can have only one set of permissions
-
-Each Custom Application has a [unique pair of user permissions](/concepts/oauth-scopes-and-user-permissions) assigned to them.
+<Warning>
```

Comment:
```suggestion
<Info>
```

Reply:
We try to avoid the use of 'warning' unless it's something really dangerous that we're trying to warn against. :)